docs/docs-ref-autogen/overview/toc.yml

```
@@ -17,3 +17,7 @@ items:
         href: /javascript/api/word
       - name: Common APIs
         href: /javascript/api/office
+  - name: Manifest reference
+    href: /javascript/api/manifest
+  - name: Manifest reference
+    href: /javascript/api/manifest
```

Seems like this should be the Requirement sets link?
```suggestion
  - name: Requirement sets
    href: /javascript/api/requirement-sets
```

---

Good catch. I think that proves this file is no longer necessary :-)